tox.ini

```
@@ -34,6 +34,20 @@ commands =
     isort {[vars]all_path}
     black {[vars]all_path}
 
+[testenv:render-k8s]
+description = Render the k8s charm
+allowlist_externals = cp
+commands =
+    cp src/k8s_charm.py src/charm.py
+    cp k8s_metadata.yaml metadata.yaml
```

By default, charmcraft includes everything under `src/`. May want to exclude. Not critical.

Iirc, this won't readily work from other dirs, unless
```suggestion
    cp {toxinidir}/src/k8s_charm.py {toxinidir}/src/charm.py
    cp {toxinidir}/k8s_metadata.yaml {toxinidir}/metadata.yaml
```